macros/ui/niceTables.pl

```diff
@@ -24,7 +29,7 @@ =head1 niceTables.pl
 relation between content cells within a column or within a row. If the
 answer is no in both cases, it is likely a case for C<LayoutTable()>.
 
-=head2 Description
+
```

Comment:
I just noticed this.  The `=head2 Description` line was replaced with an empty line, leaving this with three consecutive empty lines.  There should only be one.

Reply:
I went ahead and fixed this.